### PR TITLE
Explain our access requirements

### PR DESCRIPTION
**Pull request by mirons-google**
This PR clarifies and 'abstracts' our IAM role requirements along with the required APIs. It also created a new guide on best practices and explains in details why we need certain service accounts and access.

### CODE REVIEW

File: _includes/_howto/deployment_prerequisites.md [hunk 1]
```diff
@@ -51,25 +52,42 @@ To create a service account for Forseti Inventory, Scanner, and Enforcer, follow
...
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/storage.admin
+      --role=roles/cloudsql.viewer
+      ```
+      
+      **Project level bindings**
+      ```bash
+      $ gcloud organizations add-iam-policy-binding FORSETI_PROJECT_ID \
```
Comment (carise) [inline _includes/_howto/deployment_prerequisites.md]: I dont think this is the right gcloud command. It should probably be

gcloud projects add-iam-policy-binding ...

https://cloud.google.com/sdk/gcloud/reference/projects/add-iam-policy-binding

File: _includes/_howto/deployment_prerequisites.md [hunk 2]
```diff
@@ -51,25 +52,42 @@ To create a service account for Forseti Inventory, Scanner, and Enforcer, follow
...
+      --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
+      --role=roles/storage.objectViewer
+      ```
+      ```bash
+      $ gcloud organizations add-iam-policy-binding FORSETI_PROJECT_ID \
```
Comment (carise) [inline _includes/_howto/deployment_prerequisites.md]: same

File: _includes/_howto/deployment_prerequisites.md [hunk 3]
```diff
@@ -51,25 +52,42 @@ To create a service account for Forseti Inventory, Scanner, and Enforcer, follow
...
+      --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
+      --role=roles/storage.objectCreator
+      ```
+      ```bash
+      $ gcloud organizations add-iam-policy-binding FORSETI_PROJECT_ID \
```
Comment (carise) [inline _includes/_howto/deployment_prerequisites.md]: same

File: _docs/guides/forseti-security-best-practices.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+---
+title: Forseti Security Best Practices 
+order: 004
+---
+# {{ page.title }}
+
+## Service Accounts
+When you use multiple service accounts with your Forseti Security deployments,
```
Comment (carise) [inline _docs/guides/forseti-security-best-practices.md]: I'm not so sure that it's only the multiple service accounts that implements least privilege -- we also must observe the principle of assigning the lowest role necessary for the task. Pinging @ahoying for guidance on how to word this.

File: _docs/guides/forseti-security-best-practices.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+[published](https://cloudplatform.googleblog.com/2017/07/help-keep-your-Google-Cloud-service-account-keys-safe.html)
+best practices on least privilege, secure storage, and rotation.
+
+## Permission explanation
+It's important to know that if you don't plan to execute a particular piece of
+Forseti, e.g. `forseti_enforcer` you don't need to create that service account
+or grant those permissions.
+
+Learn more about GCP IAM roles
+[here](https://cloud.google.com/iam/docs/understanding-roles#predefined_roles).
```
Comment (carise) [inline _docs/guides/forseti-security-best-practices.md]: should we remove the anchor link (ie. remove `#predefined_roles`)?

File: _docs/guides/forseti-security-best-practices.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+### Forseti Security Service Account
+
+Forseti Security needs the following roles for `forseti_inventory` and/or
+`forseti_scanner` purposes.
+
+{% include _global/required-roles.md %}
+
+### Forseti Security GSuite Groups Inventory Service Account
+To inventory GSuite Groups and their members, Forseti Security needs access to
```
Comment (carise) [inline _docs/guides/forseti-security-best-practices.md]: How about, 

... Forseti Security uses a service account enabled for Domain Wide Delegation. The only permission this service account needs is ...

The reason I think it should be worded this way is that Forseti itself doesn't need the GSuite client id directly (we don't use it anywhere in the code).

File: _includes/_global/required-apis.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+ * **Cloud SQL API:** `sql-component.googleapis.com`
```
Comment (carise) [inline _includes/_global/required-apis.md]: Should we alpha sort these?

File: _includes/_howto/deployment_prerequisites.md (hunk 1 above)
Comment (mirons-google) [inline _includes/_howto/deployment_prerequisites.md]: Done, nice catch, thanks!

File: _includes/_howto/deployment_prerequisites.md (hunk 2 above)
Comment (mirons-google) [inline _includes/_howto/deployment_prerequisites.md]: done!

File: _includes/_howto/deployment_prerequisites.md (hunk 3 above)
Comment (mirons-google) [inline _includes/_howto/deployment_prerequisites.md]: done!

File: _docs/guides/forseti-security-best-practices.md (hunk 5 above)
Comment (mirons-google) [inline _docs/guides/forseti-security-best-practices.md]: I thought that was most relevant to the point of discussion in this doc.

File: _docs/guides/forseti-security-best-practices.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+order: 004
+---
+# {{ page.title }}
+
+## Service Accounts
+When you use multiple service accounts with your Forseti Security deployments,
+you fulfill the security best practice of least privilege. Following are the
+scenarios for which it's best to use separate service accounts:
+
+ * **Forseti Security Service Account** (required): Used by all core modules of
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: I would clarify this as:
```Forseti Security GCP Service Account```

File: _docs/guides/forseti-security-best-practices.md (hunk 6 above)
Comment (mirons-google) [inline _docs/guides/forseti-security-best-practices.md]: done!

File: _docs/guides/forseti-security-best-practices.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+the program to provide basic inventory, scanning, and enforcement actions.
+ * **GSuite Groups Inventory Service Account** (optional): Used to inventory
+GSuite Groups and their members.
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: Would suggest to include more generalization in the description, something like ```GSuite integration```.

File: _docs/guides/forseti-security-best-practices.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+### Forseti Security Service Account
+This service account is used by core modules of the Forseti service. For
+example, `forseti_inventory` uses this service account to read and store the
+supported resources. It's also used by `forseti_scanner` to scan
+`forseti_inventory` records.
+
+A good name for this service account would be `forseti-security`.
+
+### GSuite Groups Inventory Service Account
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: Would change the title of this section to be:
```Service Account for GSuite Integration```

File: _docs/guides/forseti-security-best-practices.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+It's best to use a separate service account for GSuite Groups inventory for
+privilege separation because the service account key must be local to
+`forseti_inventory`. By using a separate service account, the key scope is
+limited to GSuite Groups if the machine is compromised.
+
+If you [enable]{% link _docs/howto/gsuite-group-collection.md %})
+GSuite Group inventory and create a service account, a good name
+for the service account would be `forseti-security-gsuite-dwd`.
+
+### IAM Explain Service Account
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: Would change the title of this section to be:
```Service Account for IAM Explain```

File: _docs/guides/forseti-security-best-practices.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+the program to provide basic inventory, scanning, and enforcement actions.
+ * **GSuite Groups Inventory Service Account** (optional): Used to inventory
+GSuite Groups and their members.
+ * **IAM Explain Service Account** (optional): Used to provide
+IAM explain functionality.
+
+When you name service accounts, it's best to use a descriptive name like
+`forseti-security` or `forseti-security-gsuite`.
+
+### Forseti Security Service Account
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: Would change the title of this section to be:
```Service Account for Forseti Security```

File: _docs/guides/forseti-security-best-practices.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+since the `explain` service is an interactive tool and runs on its own GCE
+instance, it's best to apply least privilege by creating a separate service
+account for IAM Explain.
+
+If you enable IAM Explain and create a service account, a good name for the
+service account would be `forseti-security-iam-explain`.
+
+## Key security
+Whether you install and deploy Forseti manually or by using the setup wizard,
+you’ll need to store the keys securely. Google has
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: nit: Describe the key as "service account key"

File: _docs/guides/forseti-security-best-practices.md [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+### IAM Explain Service Account
+You can use the Forseti Security Service account for IAM Explain. However,
+since the `explain` service is an interactive tool and runs on its own GCE
+instance, it's best to apply least privilege by creating a separate service
+account for IAM Explain.
+
+If you enable IAM Explain and create a service account, a good name for the
+service account would be `forseti-security-iam-explain`.
+
+## Key security
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: nit: name this section as ```Service Account Key Security``` or ```Secure Service Account Keys```.

File: _docs/guides/forseti-security-best-practices.md [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+Whether you install and deploy Forseti manually or by using the setup wizard,
+you’ll need to store the keys securely. Google has
+[published](https://cloudplatform.googleblog.com/2017/07/help-keep-your-Google-Cloud-service-account-keys-safe.html)
+best practices on least privilege, secure storage, and rotation.
+
+## Permission explanation
+It's important to know that if you don't plan to execute a particular piece of
+Forseti, e.g. `forseti_enforcer` you don't need to create that service account
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: nit: we should drop "service account" here, as there is only one service account for Forseti and it is needed

File: _docs/guides/forseti-security-best-practices.md [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
+[here](https://cloud.google.com/iam/docs/understanding-roles#predefined_roles).
+
+### Forseti Security Service Account
+
+Forseti Security needs the following roles for `forseti_inventory` and/or
+`forseti_scanner` purposes.
+
+{% include _global/required-roles.md %}
+
+### Forseti Security GSuite Groups Inventory Service Account
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: This section might be better described as:
```Permissioning Service Account for GSuite Integration```

File: _includes/_howto/deployment_prerequisites.md [hunk 17]
```diff
@@ -39,10 +38,12 @@ To create a service account for Forseti Inventory, Scanner, and Enforcer, follow
       $ export GOOGLE_APPLICATION_CREDENTIALS=SERVICE_ACCOUNT_KEY_PATH
       ``` 
           
-  where `SERVICE_ACCOUNT_KEY_PATH` is the path to the json service account key you just downloaded.
+  where `SERVICE_ACCOUNT_KEY_PATH` is the path to the json service account key
```
Comment (blueandgold) [inline _includes/_howto/deployment_prerequisites.md]: We don't need to use, thus download, the json key of the forseti GCP service account anymore.  On GCP, this service account will just use the environment permission.

File: _docs/guides/forseti-security-best-practices.md [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+---
+# {{ page.title }}
+
+## Service Accounts
+When you use multiple service accounts with your Forseti Security deployments,
+you fulfill the security best practice of least privilege. Following are the
+scenarios for which it's best to use separate service accounts:
+
+ * **Forseti Security Service Account** (required): Used by all core modules of
+the program to provide basic inventory, scanning, and enforcement actions.
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: In addition to the naming, would include "GCP" in the description here.  Perhaps:
```provide basic inventory, scanning, and enforcement actions in GCP.```

File: _docs/guides/forseti-security-best-practices.md (hunk 16 above)
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: This section might be better described as:
```Permission Explanation for Forseti Security GSuite Service Account```

File: _docs/guides/forseti-security-best-practices.md (hunk 12 above)
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: This section might be better as:
```Service Account for Forseti Security```

File: _docs/guides/forseti-security-best-practices.md (hunk 10 above)
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: This section might be better as:
```Service Account for GSuite Integration```

File: _docs/guides/forseti-security-best-practices.md (hunk 11 above)
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: This section might be better as:
```Service Account for IAM Explain```

File: _docs/guides/forseti-security-best-practices.md (hunk 14 above)
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: This section might be better described as:
```Service Account Key Security```
```Security for Service Account Keys```

File: _docs/guides/forseti-security-best-practices.md (hunk 12 above)
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: Would also change the naming in above.

File: _docs/guides/forseti-security-best-practices.md [hunk 19]
```diff
@@ -0,0 +1,75 @@
...
+When you name service accounts, it's best to use a descriptive name like
+`forseti-security` or `forseti-security-gsuite`.
+
+### Forseti Security Service Account
+This service account is used by core modules of the Forseti service. For
+example, `forseti_inventory` uses this service account to read and store the
+supported resources. It's also used by `forseti_scanner` to scan
+`forseti_inventory` records.
+
+A good name for this service account would be `forseti-security`.
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: Perhaps: ```forseti-security-gcp``` would help to to disambiguate from the one for gsuite.

File: _docs/guides/forseti-security-best-practices.md [hunk 20]
```diff
@@ -0,0 +1,75 @@
...
+
+### GSuite Groups Inventory Service Account
+It's best to use a separate service account for GSuite Groups inventory for
+privilege separation because the service account key must be local to
+`forseti_inventory`. By using a separate service account, the key scope is
+limited to GSuite Groups if the machine is compromised.
+
+If you [enable]{% link _docs/howto/gsuite-group-collection.md %})
+GSuite Group inventory and create a service account, a good name
+for the service account would be `forseti-security-gsuite-dwd`.
```
Comment (blueandgold) [inline _docs/guides/forseti-security-best-practices.md]: nit: seems better without the ```-dwd``` in the naming

File: _includes/_global/required-roles.md [hunk 21]
```diff
@@ -0,0 +1,25 @@
...
+ * `roles/servicemanagement.quotaViewer`
+ * `roles/cloudsql.viewer`
+ 
+ `forseti_enforcer` requires:
+ 
+ * `roles/compute.securityAdmin`
+
+**Granted on the project where Forseti Security is deployed**
+
+ `forseti_inventory` and `forseti_scanner` require:
```
Comment (ahoying) [inline _includes/_global/required-roles.md]: enforcer will also require these for reading firewall policies from GCS and writing rollback snapshot policies to GCS (soon).

File: _includes/_howto/deployment_prerequisites.md [hunk 22]
```diff
@@ -51,25 +52,52 @@ To create a service account for Forseti Inventory, Scanner, and Enforcer, follow
...
+      --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
+      --role=roles/cloudsql.viewer
       ```
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/resourcemanager.folderAdmin
+      --role=roles/cloudsql.viewer
       ```
       ```bash
```
Comment (ahoying) [inline _includes/_howto/deployment_prerequisites.md]: Should we call this out as optional, and if they aren't using enforcer to not include it?

File: _includes/_global/required-roles.md (hunk 21 above)
Comment (mirons-google) [inline _includes/_global/required-roles.md]: done!

File: _includes/_howto/deployment_prerequisites.md (hunk 22 above)
Comment (mirons-google) [inline _includes/_howto/deployment_prerequisites.md]: Is it really optional? I didn't think so. @blueandgold can you help confirm?

Comment (blueandgold) [inline _includes/_howto/deployment_prerequisites.md]: Is this about ```roles/compute.securityAdmin```?  (sorry github ui can be hard to follow)
If so, there is already ```roles/iam.securityReviewer``` above, so ```roles/compute.securityAdmin``` can be optional, and scoped to Enforcer.

File: _docs/guides/best-practices.md [hunk 23]
```diff
@@ -0,0 +1,84 @@
...
+
+### Forseti Security Service Account
+This service account is used by core modules of the Forseti service. For
+example, `forseti_inventory` uses this service account to read and store the
+supported resources. It's also used by `forseti_scanner` to scan
+`forseti_inventory` records.
+
+A good name for this service account would be `forseti-security`.
+
+### GSuite Groups Service Account
```
Comment (blueandgold) [inline _docs/guides/best-practices.md]: nit: drop the mentioning of ```Groups``` and just leave it at ```GSuite```

File: _docs/guides/best-practices.md [hunk 24]
```diff
@@ -0,0 +1,84 @@
...
+It's best to use a separate service account for GSuite Groups inventory for
+privilege separation because the service account key must be local to
+`forseti_inventory`. By using a separate service account, the key scope is
+limited to GSuite Groups if the machine is compromised.
+
+If you [enable]{% link _docs/howto/gsuite-group-collection.md %})
+GSuite Group inventory and create a service account, a good name
+for the service account would be `forseti-security-gsuite-dwd`.
```
Comment (blueandgold) [inline _docs/guides/best-practices.md]: Would drop ```dwd``` in the naming.

File: _docs/guides/best-practices.md [hunk 25]
```diff
@@ -0,0 +1,84 @@
...
+Learn more about GCP IAM roles
+[here](https://cloud.google.com/iam/docs/understanding-roles#predefined_roles).
+
+### Service Account for Forseti Security
+Forseti Security needs the following roles for `forseti_inventory` and/or
+`forseti_scanner` purposes.
+
+{% include _global/required-roles.md %}
+
+### Service Account for GSuite Groups
```
Comment (blueandgold) [inline _docs/guides/best-practices.md]: ditto: drop ```Groups```

File: _includes/_global/required-apis.md [hunk 26]
```diff
@@ -0,0 +1,6 @@
+ * **Admin SDK API:** `admin.googleapis.com`
+ * **Cloud Resource Manager API:** `cloudresourcemanager.googleapis.com`
+ * **Cloud SQL Admin API:** `sqladmin.googleapis.com`
+ * **Cloud SQL API:** `sql-component.googleapis.com`
+ * **Compute Engine API**: `compute-component.googleapis.com`
+ * **Deployment Manager API:** `deploymentmanager.googleapis.com`
```
Comment (blueandgold) [inline _includes/_global/required-apis.md]: Can we add Andrews explanation of how the required API's are determined?  Andrew and I think it's a good idea.
```
It depends on where quota is calculated. If it is on the caller project then the API needs to be enabled on the Forseti project, if it is on the called project, then it is only needed there.
```

File: _docs/guides/best-practices.md [hunk 27]
```diff
@@ -0,0 +1,84 @@
...
+
+## Service Accounts
+When you use multiple service accounts with your Forseti Security deployments,
+you fulfill the security best practice of least privilege. Following are the
+scenarios for which it's best to use separate service accounts:
+
+ * **[Forseti Security Service Account](#forseti-security-service-account)**
+ (required): Used by all core modules of the program to provide basic
+ inventory, scanning, and enforcement actions.
+ * **[GSuite Groups Service Account](#gsuite-groups-service-account)**
```
Comment (mirons-google) [inline _docs/guides/best-practices.md]: I disagree here, I think it's important to note that this account's access is scoped only the Groups API. That should be reflected in the name and in the docs that way.

File: _docs/guides/best-practices.md (hunk 23 above)
Comment (mirons-google) [inline _docs/guides/best-practices.md]: See above.

File: _docs/guides/best-practices.md (hunk 24 above)
Comment (mirons-google) [inline _docs/guides/best-practices.md]: To my point and per the discussion above I think it would be better to s/dwd/groups/

**Comment by blueandgold** [comment]
One more note: please sync the recommended service account namings here, with the defaults in the setup wizard.  :)

File: _docs/guides/best-practices.md [hunk 28]
```diff
@@ -0,0 +1,84 @@
...
+ (optional): Used to inventory GSuite Groups and their members.
+ Forseti IAM explain requires this to be enabled.
+ * **[Forseti Explain Service Account](#forseti-explain-service-account)**
+ (optional): Used to provide IAM explain functionality.
+
+When you name service accounts, it's best to use a descriptive name like
+`forseti-security` or `forseti-security-gsuite`.
+
+### Forseti Security Service Account
```
Comment (felixbb) [inline _docs/guides/best-practices.md]: Out of scope for this change but I'd encourage you to rethink if this authority can be combined in a single service account. If what we want to achieve is least privilege, having less service accounts is almost as important to granting the 'boxes' less privileges.

File: _docs/guides/best-practices.md [hunk 29]
```diff
@@ -0,0 +1,84 @@
...
+or grant those permissions.
+
+Learn more about GCP IAM roles
+[here](https://cloud.google.com/iam/docs/understanding-roles#predefined_roles).
+
+### Service Account for Forseti Security
+Forseti Security needs the following roles for `forseti_inventory` and/or
+`forseti_scanner` purposes.
+
+{% include _global/required_roles.md %}
```
Comment (felixbb) [inline _docs/guides/best-practices.md]: Why do we need all these roles for crawling? This should be aggregated inside one curated role. Maybe talk to the IAM integration team, explain your case and ask them if they agree that this should be under securityReviewer so we can potentially add the required permissions. Or maybe we have to introduce a curated role 'inventoryCreator' or something like that. This list seems excessive.

File: _docs/guides/best-practices.md [hunk 30]
```diff
@@ -0,0 +1,84 @@
+---
+title: Best Practices
```
Comment (felixbb) [inline _docs/guides/best-practices.md]: I think it would be helpful to provide some scenarios that we've seen with deployments. For example 'inventory with IAM Explain' - what is the actual recommendation of such a deployment. How about 'inventory + scanner' etc. The current way the document is setup there is still quite some transfer thinking required.

Comment (felixbb) [inline _docs/guides/best-practices.md]: I do think though that it is a great improvement over what we had before, hence approved.

File: _docs/guides/best-practices.md (hunk 28 above)
Comment (mirons-google) [inline _docs/guides/best-practices.md]: What's unique about GSuite intengration is that it requries the key to be local to the code (on the same machine). The GCP APIs can use default credentials (can't be used with the GSuite Admin SDK).

Thus the need for a separation of duties. Is there a better term here to use instead? Am I not clear in the description as to why? Maybe the concern shouldn't be a concern.

File: _docs/guides/best-practices.md (hunk 30 above)
Comment (mirons-google) [inline _docs/guides/best-practices.md]: Thanks just so I understand you're proposing a new section (different scope) regarding what's recommended on common deployment scenarios?

File: _docs/quickstarts/explain/index.md [hunk 31]
```diff
@@ -1,5 +1,5 @@
 ---
-title: IAM Explain
+title: Explain
```
Comment (Tabrarian) [inline _docs/quickstarts/explain/index.md]: Felix M had indicated to me that the "official" product name would be IAM Explain; can we confirm?

File: _docs/quickstarts/forseti-security/index.md [hunk 32]
```diff
@@ -77,24 +77,16 @@ file based on your input.
...
-  account:
-      - `roles/browser`
-      - `roles/compute.networkAdmin`
-      - `roles/editor`
-      - `roles/iam.securityReviewer`
-      - `roles/resourcemanager.folderAdmin`
-      - `roles/storage.admin`
+  1. Next, the setup wizard automatically assigns the required roles to the GCP service
+  account. The [Forseti Security Best Practices Guide]({% link _docs/guides/best-practices.md %})
+  has more detail on the required roles.
```
Comment (Tabrarian) [inline _docs/quickstarts/forseti-security/index.md]: "For details about the required roles, see {best practices link}"

File: _docs/quickstarts/index.md [hunk 33]
```diff
@@ -24,7 +24,7 @@ violations.
 
 Setting up Enforcer to automatically correct policy discrepancies.
 
-**[IAM Explain]({% link _docs/quickstarts/explain/index.md %})**
+**[Explain]({% link _docs/quickstarts/explain/index.md %})**
```
Comment (Tabrarian) [inline _docs/quickstarts/index.md]: See above, re: product name.

File: _includes/_howto/deployment_prerequisites.md [hunk 34]
```diff
@@ -51,25 +46,51 @@ To create a service account for Forseti Inventory, Scanner, and Enforcer, follow
...
+      --role=roles/servicemanagement.quotaViewer
       ```
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/storage.admin
+      --role=roles/cloudsql.viewer
+      ```
+      
+      **It should be noted that binding at any other level than the Organization
```
Comment (Tabrarian) [inline _includes/_howto/deployment_prerequisites.md]: "Note that binding at any other level..."

Comment (mirons-google) [inline _includes/_howto/deployment_prerequisites.md]: done

File: _docs/quickstarts/index.md (hunk 33 above)
Comment (mirons-google) [inline _docs/quickstarts/index.md]: reverted

File: _docs/quickstarts/forseti-security/index.md (hunk 32 above)
Comment (mirons-google) [inline _docs/quickstarts/forseti-security/index.md]: done

File: _docs/quickstarts/explain/index.md (hunk 31 above)
Comment (mirons-google) [inline _docs/quickstarts/explain/index.md]: reverted for now.

**Comment by mirons-google** [comment]
@Tabrarian @carise PTAL

**Comment by mirons-google** [comment]
Thanks everyone!

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as all commit authors are covered by CLA.
